bedrock/firefox/templates/firefox/compare/chrome.html

```diff
@@ -282,38 +287,55 @@ <h2>{{ _('<strong>Portability</strong>') }}</h2>
           </tbody>
         </table>
 
-      <p class="compare-content">
+      <p>
         {% trans %}
         Almost needless to say, versions of both Firefox and Chrome are available for the most popular desktop and mobile operating systems (Windows, macOS, Linux, Android, iOS).
         {% endtrans %}
       </p>
 
-      <p class="compare-content">
+      <p>
         {% trans %}
         Both Chrome and Firefox also allow you to sync things like passwords, bookmarks, and open tabs across all your devices. If you have a Firefox account, you can manually send an open tab on your desktop to your mobile device or vice versa. With Chrome, it’s done automatically if you’ve chosen that setting in your preferences. Not having to manually send the tab from one device to the other is convenient when you want to do something like continue reading an article you didn’t finish earlier. But there could be times where automatic syncing might not be ideal if there’s a chance multiple users are browsing while signed in to your Google account.
         {% endtrans %}
       </p>
 
+      {% include '/firefox/compare/includes/download-menu-list.html' %}
+
     </section>
 
     <section class="compare-section-overall">
       <header class="compare-section-header">
         <h2>{{ _('Overall Assessment') }}</h2>
       </header>
 
-      <p class="compare-content">
+      <p>
         {% trans %}
         We think it’s fair to say Firefox and Chrome are really neck and neck in terms of portability and utility, with Chrome having a slight edge in utility because of its huge library of extensions and add-on features. But in terms of Privacy, Firefox wins the day with our commitment to preserving our users’ online data and providing free baked-in services like password managers that also alert you if there happens to be a data breach involving your credentials.
         {% endtrans %}
       </p>
 
-      <p class="compare-content">
+      <p>
         {% trans %}
         For practical purposes, there’s obviously really nothing preventing you from using both browsers—Firefox for those moments in life when privacy really matters, and Chrome if you’re still invested in the Google ecosystem. Yet with the growing number of incursions into our personal data, Firefox may prove to be the right choice in the long run for those of us who value protecting our personal privacy online.
         {% endtrans %}
       </p>
 
+      <p class="compare-version-disclaimer">
+        {% trans %}
+        The comparisons made here were done so across browser release versions as follows:<br> Firefox (70) | Chrome (74)
+        {% endtrans %}
+      </p>
+
+      <aside class="bottom-download-cta">
+        {{ download_firefox(dom_id='download-bottom', alt_copy=_('Download Firefox'), download_location='bottom cta') }}
```

Comment:
I think our convention here is `secondary cta`?
```suggestion
        {{ download_firefox(dom_id='download-bottom', alt_copy=_('Download Firefox'), download_location='secondary cta') }}
```

Reply:
This is the same for other pages.